code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -149,16 +149,22 @@
 
 /obj/item/reagent_containers/hypospray/autoinjector/attack_self(mob/user as mob)
 	if(is_open_container())
+		to_chat(user, SPAN_WARNING("\The [src] hasn't been secured yet!"))
+		return
+	if(do_after(user, 1 SECOND, TRUE))
+		inject(user, user, TRUE)
+
+/obj/item/reagent_containers/hypospray/autoinjector/AltClick(mob/user)
```

Comment:
Undocumented change, that may cause issues for those who are used to the old behavior. Document this in the changelog.